en/views/helpers/paginator.rst

```diff
@@ -403,7 +403,7 @@ Generating Pagination URLs
 By default returns a full pagination URL string for use in non-standard contexts
 (i.e. JavaScript). ::
 
-    echo $this->Paginator->generateUrl(['sort' => 'title']);
+    echo $this->Paginator->generateUrl(['?' => ['sort' => 'title']]);
```

Comment:
Should this change be mentioned in the migration guide as well? I am not sure folks will connect the Router change to pagination.

Reply:
Done